Angular InstantSearch/conditional-request/src/app/app.component.html

```diff
@@ -11,33 +11,38 @@ <h1 class="header-title">
 </header>
 
 <div class="container">
-  <div class="results">
-    <ais-instantsearch [config]="config">
-      <p>The request is only triggered once the query is not empty.</p>
-      <div class="searchbox">
-        <ais-search-box></ais-search-box>
-      </div>
-      <div class="search-panel">
-        <div class="search-panel__results">
-          <ais-hits>
-            <ng-template let-hits="hits">
-              <ol class="ais-Hits-list">
-                <li class="ais-Hits-item" *ngFor="let hit of hits">
+  <ais-instantsearch [config]="config">
+    <div class="search-panel">
+      <div class="search-panel__results">
+        <p>The request is only triggered once the query is not empty.</p>
```

Comment:
```suggestion
        <p>The request is only triggered once the query is no longer empty.</p>
```

Reply:
actually not is a better description, "no longer" implies it's only the first empty query which is skipped

Reply:
In that case it's "when" not "once".